src/internet-analyzer/azext_internet_analyzer/vendored_sdks/frontdoor/models/_front_door_management_client_enums.py

```diff
@@ -0,0 +1,318 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum
+
+
+class NetworkOperationStatus(str, Enum):
+
+    in_progress = "InProgress"
+    succeeded = "Succeeded"
+    failed = "Failed"
+
+
+class NetworkExperimentResourceState(str, Enum):
+
+    creating = "Creating"
+    enabling = "Enabling"
+    enabled = "Enabled"
+    disabling = "Disabling"
+    disabled = "Disabled"
+    deleting = "Deleting"
+
+
+class State(str, Enum):
+
+    enabled = "Enabled"
+    disabled = "Disabled"
+
+
+class AggregationInterval(str, Enum):
```

Comment:
For latency scorecards, the valid aggregation values are: Daily, Weekly, Monthly

For timeseries, the valid aggregation values are: Hourly, Daily

Reply:
See RP API definition: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json

Reply:
Ok, I see that the **LatencyScorecardAggregationInterval** and **TimeseriesAggregationInterval** are defined further down. This **AggregationInterval** class can be removed (shouldn't be used).

Reply:
well, the SDK is autogenerated from swagger as well, so we should figure out whit AggregationInterval is actually generated.